funcx_sdk/funcx/serialize/off_process_checker.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import logging
+import socket
+import argparse
+import pickle
+
+
+def server(port=0, host='', debug=False, datasize=102400):
+
+    from funcx.serialize import FuncXSerializer
+    fxs = FuncXSerializer()
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.bind((host, port))
+        bound_port = s.getsockname()[1]
+        print(f"BINDING TO:{bound_port}", flush=True)
+        s.listen(1)
+        conn, addr = s.accept()  # we only expect one incoming connection here.
+        with conn:
+            print('Connected by', addr)
```

Comment:
Do we need all these prints below `BINDING TO`?

Reply:
Removed most of these.